flink-table/flink-table-common/src/main/java/org/apache/flink/table/connector/sink/OutputFormatProvider.java

```diff
@@ -36,6 +38,23 @@ static OutputFormatProvider of(OutputFormat<RowData> outputFormat) {
 		return () -> outputFormat;
```

Comment:
It's seem this method is useless now? If no one uses it, please delete it. Otherwise, we can just treat this method as
`of(outputFormat, null)`

Reply:
@fsk119 I have rebased to master,but push failed.What should I do next.
![image](https://user-images.githubusercontent.com/31235759/101611000-ea1e3480-3a43-11eb-9931-6b93ed7aed0e.png)